src/EmailPage/index.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+import * as React from 'react';
+import css from 'styled-jsx/css';
+import { Close } from '@kiwicom/orbit-components/lib/icons';
+import image from '../../static/mailbox@2x.png';
+
+type Props = {
+  email: string,
+};
+
+const style = css`
+  .Email {
+    position: absolute;
+    right: 0;
+    width: 480px;
+    height: 100vh;
+    background-color: #ffffff;
+    box-shadow: 0 4px 7px 0 rgba(0, 0, 0, 0.15);
+    padding-top: 240px;
+  }
+  p.title {
```

Comment:
This should have a `margin-bottom: 6px` as per the design https://app.zeplin.io/project/5a8e8992554d385c76ad5aeb/screen/5aa6b8a48e193557a2446de1